\hypertarget{classWt_1_1Dbo_1_1NoUniqueResultException}{}\doxysection{Wt\+::Dbo\+::No\+Unique\+Result\+Exception Class Reference}
\label{classWt_1_1Dbo_1_1NoUniqueResultException}\index{Wt::Dbo::NoUniqueResultException@{Wt::Dbo::NoUniqueResultException}}


Exception thrown when a query unexpectedly finds a non-\/unique result.  




{\ttfamily \#include $<$Wt/\+Dbo/\+Exception.\+h$>$}



Inheritance diagram for Wt\+::Dbo\+::No\+Unique\+Result\+Exception\+:
% FIG 0


Collaboration diagram for Wt\+::Dbo\+::No\+Unique\+Result\+Exception\+:
% FIG 1
\doxysubsection*{Public Member Functions}
\begin{DoxyCompactItemize}
\item 
\mbox{\Hypertarget{classWt_1_1Dbo_1_1NoUniqueResultException_ad89a102eab9ef0d5d0d36975a6c958ab}\label{classWt_1_1Dbo_1_1NoUniqueResultException_ad89a102eab9ef0d5d0d36975a6c958ab}} 
\mbox{\hyperlink{classWt_1_1Dbo_1_1NoUniqueResultException_ad89a102eab9ef0d5d0d36975a6c958ab}{No\+Unique\+Result\+Exception}} ()
\begin{DoxyCompactList}\small\item\em Constructor. \end{DoxyCompactList}\end{DoxyCompactItemize}


\doxysubsection{Detailed Description}
Exception thrown when a query unexpectedly finds a non-\/unique result. 

This Exception is thrown by Query\+::result\+Value() when the query has more than one result. 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
wt-\/4.\+8.\+2/src/\+Wt/\+Dbo/Exception.\+h\end{DoxyCompactItemize}
